static/app/views/issueDetails/streamline/header.tsx

```diff
@@ -199,16 +204,18 @@ const HeaderGrid = styled('div')`
 `;
 
 const PrimaryTitle = styled(Tooltip)`
-  font-size: 20px;
-  font-weight: ${p => p.theme.fontWeightBold};
   overflow-x: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
+  font-size: 20px;
+  font-weight: ${p => p.theme.fontWeightBold};
+  flex-shrink: 0;
 `;
 
-const SecondaryTitle = styled(PrimaryTitle)<{isDefault: boolean}>`
-  font-size: ${p => p.theme.fontSizeMedium};
-  font-weight: ${p => p.theme.fontWeightNormal};
+const SecondaryTitle = styled(Tooltip)<{isDefault: boolean}>`
+  overflow-x: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
```

Comment:
fyi we do have a util for these styles: `${p.theme.overflowEllipsis}`

Reply:
Could we use skipWrapper on the tooltip here and keep styling the PrimaryTitle?